en/home.md

```diff
@@ -1,134 +1,168 @@
 
-## Misc {docsify-ignore}
+  You can accept payments within your app by easily embedding our Android and iOS Mobile SDK within your app. Similar to Snap, the Mobile SDK also provides drop-in user interface to accept payments using [Midtrans’s various payment methods](https://midtrans.com/payments).
 
-<!-- TODO: write this page -->
-<div class="my-card">
+  <details>
+  <summary><b>Try Mobile SDK via Android Simulator</b></summary>
+  <article>
+  <div style="text-align: center;">
+  <iframe src="https://appetize.io/embed/9r0b89zu862f8eu1ukd0ecpgxc?device=nexus5&scale=75&orientation=portrait&osVersion=8.1"width="300px" height="600px" frameborder="0" scrolling="no"></iframe>
+  </div>
+  </article>
+  </details>
 
-<!-- #### [Integrate Payment to POS](/en/pos/overview.md) -->
-#### [Integrate Payment to POS](#accept-payment-on-point-of-sales-vending-machine-iot-devices-etc)
-</div>
+  </div>
 
-<br> <br>
+  <div class="my-card">
 
 
-## Choose based on Top Business Use Cases {docsify-ignore}
+  #### [Customize Your Own Payment Page (Core API)](/en/core-api/overview.md)
 
-Here are some popular use cases that may help you choose:
+  If you need to customize the payment-flow or the user interface to fit your unique needs, use our Core API. Core API can be used for your website, web application, point of sales, IoT (Internet of Things) or any other internet-capable device, to connect with Midtrans and start accepting payments. Core API uses the familiar REST API standard with JSON-based payload.
+  </div>
 
-#### Accept payment on your e-commerce web, app
+  <div class="my-card">
 
-Accept payment from your customer straight within your website/app with Card Transaction, Bank Transfer, Direct Debit, E-Wallet, [and more](https://midtrans.com/payments) as payment methods. Use [Snap beautiful interface](/en/snap/overview.md) or [Customizable Core API](/en/core-api/overview.md) to enable your web and app to accept payment securely in a few simple steps.
 
-#### Subscription / Recurring Service
+  #### [Use CMS Payment Plugin](/en/snap/with-plugins.md)
 
-Charge customer for recurring payment. For example payment of repeat item purchase, subscription, membership, or billing with flexible interval period according to your business needs. So your customer can be automatically charged via Midtrans, without asking your customer to do manual payment each time. Recurring payment is available via [Snap](/en/snap/advanced-feature.md#recurring-subscription-card-transaction) and [Core API](/en/core-api/advanced-features.md#recurringone-click-transaction) product. \**Recurring only available for some specific payment channels*.
+  If you are already using e-commerce Content Management System such as Magento 2, WooCommerce, and so on, you can easily integrate with Midtrans payment page by installing plugin in a few simple clicks.
+  </div>
 
-#### Send Payment Invoices via Email, Link, Whatsapp, Instagram, Social Media, Messaging App, etc
```

Comment:
I saw this `via Email, Link, Whatsapp, Instagram, Social Media, Messaging App, etc` was removed from the title, can you please at least put it at the paragraph text? we'd like to keep those keywords for people searching with keyword like `whatsapp` `instagram` etc. thanks @AnujaChangede